### DISCUSSION TITLE
why full private key getting logged in logs,

### DISCUSSION THREAD

**Discussion by shaktiks** [actions/create-github-app-token#188] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/7eaba35d-2882-4d9a-a4c8-3f22086652e5)

**Comment by BrendenWalker** [comment, marked as answer]
It sounds like you added the private-key as a variable and not a secret.  This is how it looks in my logs:

``` Run actions/create-github-app-token@v1
  with:
    app-id: 11[2](https://github.com/bccworkshop/TestParent/actions/runs/12965260374/job/36164811495#step:2:2)3127
    private-key: ***

**Reply by parkerbxyz** [reply]
Yes, see [Using secrets in GitHub Actions](https://docs.github.com/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions) for instructions on how to store your private key as a secret.
